tools/topology/topology2/cavs/cavs-sdw.conf

```diff
@@ -24,7 +27,31 @@
 <pipeline.conf>
 <dai.conf>
 <host.conf>
+<dmic-default.conf>
+
+Define {
+	MCLK 38400000
```

Comment:
We should probably prefix this as it could be for SSP0, SSP1 (e.g. SSP$_MCLK) or some other DAI. Can be done in a later PR though.